components/select/index.en-US.md

```diff
@@ -0,0 +1,59 @@
+---
+category: Components
+type: Form Control
+title: Select
+---
+
+A Selector similar to Select2.
+
+## When to use
+
+Select provides a dropdown menu for choosen, for replacing origin selector, or demanding of a elegent one.
```

Comment:
A dropdown menu for choosing, an elegant alternative to the native select component.